ibis/expr/types/core.py

```diff
@@ -493,6 +493,34 @@ def to_delta(
         """
         self._find_backend(use_default=True).to_delta(self, path, **kwargs)
 
+    @experimental
+    def to_torch(
+        self,
+        *,
+        params: Mapping[ir.Scalar, Any] | None = None,
+        limit: int | str | None = None,
+        **kwargs: Any,
+    ) -> None:
```

Comment:
```suggestion
    ) -> dict[str, torch.Tensor]:
```

although if we don't want to have to install `torch` just for the type hint I get it

Reply:
I think it should be fine to just leave it in. You won't get tab completion if you don't have it installed but it shouldn't break anything to use the annotation.